Item(by='headalgorithm', descendants=49, kids=[24932961, 24933993, 24932593, 24933685, 24932598, 24937976, 24932937, 24938417, 24932480, 24936100, 24933277, 24933318], score=102, time=1603987315, title='Understanding Causality Is the Next Challenge for Machine Learning', item_type='story', url='https://spectrum.ieee.org/tech-talk/artificial-intelligence/machine-learning/understanding-causality-is-the-next-challenge-for-machine-learning', parent=None, text=None)